config/config.exs

```diff
@@ -5,3 +5,5 @@ use Mix.Config
 config :logger,
        level: :info,
        compile_time_purge_level: :debug
+
+config :json, log_level: :info
```

Comment:
Only thing left to do, is if you could kindly update the README to mention this new config option. 🙏